### PR TITLE
Added Sandboxie Classic

### PR DESCRIPTION
**Pull request by thesweepy**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24380)

### CODE REVIEW

File: manifests/s/SandboxieClassic/SandboxieClassic/5.51.3/SandboxieClassic.SandboxieClassic.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: SandboxieClassic.SandboxieClassic
+PackageVersion: 5.51.3
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.0.0
+
```
Comment (ItzLevvie) [inline manifests/s/SandboxieClassic/SandboxieClassic/5.51.3/SandboxieClassic.SandboxieClassic.yaml:9]:
```suggestion
# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json

PackageIdentifier: SandboxiePlus.SandboxieClassic
PackageVersion: 5.51.3
DefaultLocale: en-US
ManifestType: version
ManifestVersion: 1.0.0
```

File: manifests/s/SandboxieClassic/SandboxieClassic/5.51.3/SandboxieClassic.SandboxieClassic.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+PackageUrl: https://sandboxie-plus.com/
+License: GPL
+#LicenseUrl: 
+#Copyright: 
+#CopyrightUrl: 
+ShortDescription: Open Source sandbox-based isolation software
+#Description: 
+ManifestType: defaultLocale
+ManifestVersion: 1.0.0
+
```
Comment (ItzLevvie) [inline manifests/s/SandboxieClassic/SandboxieClassic/5.51.3/SandboxieClassic.SandboxieClassic.locale.en-US.yaml:22]:
```suggestion
# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json

PackageIdentifier: SandboxiePlus.SandboxieClassic
PackageVersion: 5.51.3
PackageLocale: en-US
Publisher: sandboxie-plus.com
PublisherUrl: https://xanasoft.com
PublisherSupportUrl: https://github.com/sandboxie-plus/Sandboxie/issues
Author: https://xanasoft.com
PackageName: Sandboxie Classic
PackageUrl: https://github.com/sandboxie-plus/Sandboxie
License: GNU General Public License v3.0
LicenseUrl: https://raw.githubusercontent.com/sandboxie-plus/Sandboxie/master/LICENSE
CopyrightUrl: https://raw.githubusercontent.com/sandboxie-plus/Sandboxie/master/LICENSE
ShortDescription: Sandboxie is a sandbox-based isolation software for 32- and 64-bit Windows NT-based operating systems.
Description: Sandboxie is a sandbox-based isolation software for 32- and 64-bit Windows NT-based operating systems. It was developed by Sophos (which acquired it from Invincea, which acquired it earlier from the original author Ronen Tzur). It creates a sandbox-like isolated operating environment in which applications can be run or installed without permanently modifying the local or mapped drive. Such an isolated virtual environment allows controlled testing of untrusted programs and web surfing.
Moniker: sandboxie-classic
Tags:
- sandbox
ManifestType: defaultLocale
ManifestVersion: 1.0.0
```

File: manifests/s/SandboxieClassic/SandboxieClassic/5.51.3/SandboxieClassic.SandboxieClassic.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#  Scope: 
+  InstallerUrl: https://github.com/sandboxie-plus/Sandboxie/releases/download/0.9.3/Sandboxie-Classic-x64-v5.51.3.exe
+  InstallerSha256: 919E941C5B3C9DB91C34BA6DD41B75C1A281DD1B09F7E98F728B4C4F5AE8C761
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
+  UpgradeBehavior: install
+ManifestType: installer
+ManifestVersion: 1.0.0
+
```
Comment (ItzLevvie) [inline manifests/s/SandboxieClassic/SandboxieClassic/5.51.3/SandboxieClassic.SandboxieClassic.installer.yaml:20]:
```suggestion
# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json

PackageIdentifier: SandboxiePlus.SandboxieClassic
PackageVersion: 5.51.3
MinimumOSVersion: 10.0.0.0
InstallModes:
- interactive
- silent
- silentWithProgress
Installers:
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: inno
  Scope: machine
  InstallerUrl: https://github.com/sandboxie-plus/Sandboxie/releases/download/0.9.3/Sandboxie-Classic-x64-v5.51.3.exe
  InstallerSha256: 919e941c5b3c9db91c34ba6dd41b75c1a281dd1b09f7e98f728b4c4f5ae8c761
  UpgradeBehavior: install
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: inno
  Scope: machine
  InstallerUrl: https://github.com/sandboxie-plus/Sandboxie/releases/download/0.9.3/Sandboxie-Classic-x86-v5.51.3.exe
  InstallerSha256: 427310f9e6bfe2ba8376f39ef5be929cfb7a83d25db89c9b99b49358562f0545
  UpgradeBehavior: install
ManifestType: installer
ManifestVersion: 1.0.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Move and rename all of the files from `SandboxieClassic.SandboxieClassic` to `SandboxiePlus.SandboxieClassic`.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Use the metadata from https://github.com/microsoft/winget-pkgs/pull/24380#pullrequestreview-726813932

File: manifests/s/SandboxiePlus/SandboxieClassic/5.51.3/SandboxiePlus.SandboxieClassic.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+PackageVersion: 5.51.3
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: inno
```
Comment (ItzLevvie) [inline manifests/s/SandboxiePlus/SandboxieClassic/5.51.3/SandboxiePlus.SandboxieClassic.installer.yaml:14]:
```suggestion
  InstallerType: nullsoft
```

File: manifests/s/SandboxiePlus/SandboxieClassic/5.51.3/SandboxiePlus.SandboxieClassic.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  Scope: machine
+  InstallerUrl: https://github.com/sandboxie-plus/Sandboxie/releases/download/0.9.3/Sandboxie-Classic-x64-v5.51.3.exe
+  InstallerSha256: 919e941c5b3c9db91c34ba6dd41b75c1a281dd1b09f7e98f728b4c4f5ae8c761
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: inno
```
Comment (ItzLevvie) [inline manifests/s/SandboxiePlus/SandboxieClassic/5.51.3/SandboxiePlus.SandboxieClassic.installer.yaml:21]:
```suggestion
  InstallerType: nullsoft
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Oops, since your original commit had `/S` try it with nullsoft which will automatically run the installer with `/S` for silent/unattended mode.

If it still fails, I'll take a look at it later.

**Review by OfficialEsco — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**